### ISSUE TITLE
ZBL    CNtable.txt could not be found

### ISSUE THREAD

**Issue by MingjieWen** [deepmodeling/deepmd-kit#3963]
### Summary

I recently encountered a problem when using DeepMD-kit. When I tried to add the ZBL item in run_param.json, I encountered the problem that CNtable.txt could not be found. I made sure that this file existed in the corresponding location. I would like to ask how to solve this problem and whether I need to modify other places(such as mechine.json)?

### DeePMD-kit Version

v2.1.5

### Backend and its version

python3.10

### Details

<img width="163" alt="input" src="https://github.com/deepmodeling/deepmd-kit/assets/102695199/8bfb1fd6-d575-41d8-b95f-17847247ecb4">
<img width="416" alt="error" src="https://github.com/deepmodeling/deepmd-kit/assets/102695199/fa1df231-f2db-4cc0-bdf4-cc0f378b8ab6">

**Comment by njzjz** [comment]
Could you provide an example for reproducing?

**Comment by MingjieWen** [comment]
Hello, jinzhe. I added zbl potential in the DPGEN process. The DPGEN file is too large. Is there any additional solution? I can manually submit the calculation file of the model training stage. I have built the input, but I don’t know how to submit this type of task. The sub and sub.run files are as follows
[001.zip](https://github.com/user-attachments/files/16171209/001.zip)

**Comment by njzjz** [comment]
There are 4 types so the tabulate file should contain $\binom{2}{4} =12$ columns.

> I added zbl potential in the DPGEN process.

Do you face this problem only with DP-GEN or also without DP-GEN?

**Comment by MingjieWen** [comment]
Yes, only with DP-GEN, there should be 4 folders. I am trying to submit each file separately and use manual method in train 01 stage, I wonder if it is feasible

**Comment by njzjz** [comment]
> Yes, only with DP-GEN

I transfer this issue to DP-GEN. Could you provide more information about DP-GEN?
